### PR TITLE
Fix go toolchain

### PR DESCRIPTION
**Pull request by pavel-the-best**

### CODE REVIEW

**Comment by MikailBag** [comment]
Please, rebase on master and resolve merge conflicts.

File: toolchains/go/detect.sh [hunk 1]
```diff
@@ -1,13 +1,21 @@
 #!/usr/bin/env bash
-if [[ "x$JJS_GO_SKIP" != "x" ]]; then
-    exit 1;
-fi
+
+versionlt() {
+    [ "$1" = "`printf '%s\n%s' $1 $2 | sort -V | head -n1`" ]
+}
+
 go_path=$( command -v go )
```
Comment (sleirsgoevy) [inline toolchains/go/detect.sh]:
```
- go_path=$( command -v go )
+ go_path="$( command -v go )"
```

File: toolchains/go/invoke-conf.yaml [hunk 2]
```diff
@@ -5,6 +5,10 @@ filename: "source.go"
 build-limits:
   time: 6000
 
+# TODO disable this hack
+env-passing: true
```
Comment (MikailBag) [inline toolchains/go/invoke-conf.yaml]: Is it required?

**Comment by pavel-the-best** [comment]
Moved to https://github.com/jjs-dev/jjs/pull/315

**Pull request closed without merging**